lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -319,20 +354,12 @@ getTransaction n = label "getTransaction" $ do
   where
     getWitness :: Get TxWitness
     getWitness = do
-        tag <- getTxWitnessTag
-        let len = txWitnessSize tag
+        tag <- lookAhead getTxWitnessTag
+        let len = txWitnessSize tag + txWitnessTagSize
```

Comment:
The tag is now part of the bytes themselves (cf utxoWitness and legacyUtxoWitness)